specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/WebApps.json

```diff
@@ -2872,6 +2872,108 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/deploymentStatus": {
+      "get": {
+        "tags": [
+          "WebApps"
+        ],
+        "summary": "List deployment statuses for an app (or deployment slot, if specified).",
+        "operationId": "WebApps_ListProductionSiteDeploymentStatuses",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of the app.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/CsmDeploymentStatusCollection"
+            }
+          },
+          "default": {
+            "description": "App Service error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List Deployment Status": {
+            "$ref": "./examples/ListSiteDeploymentStatus.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/deploymentStatus/{operationId}": {
```

Comment:
>operationId [](http://example.com/codeflow?start=127&length=11)

Can you please rename to something like deploymentstatusId

Reply:
Fixed